### PR TITLE
Add OGO intake

### PR DESCRIPTION
**Pull request by Sambogo101**
Hello, this PR adds OGO intake format.

### CODE REVIEW

**Review by squioc — CHANGES_REQUESTED** [review]
Thanks for your contribution.

Could you add some [tests](https://github.com/SEKOIA-IO/intake-formats/blob/main/doc/testing.md), a [manifest file](https://github.com/SEKOIA-IO/intake-formats/blob/main/doc/format.md#format-_meta-directory) and a logo for the format?

**Comment by Sambogo101** [comment]
@squioc tests adjusted with new module name. My bad.

File: OGO/ogo-shield/_meta/manifest.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+uuid: cf5c916e-fa26-11ed-a844-f7f4d7348199
+name: OGO Shield
+slug: ogo-shield
+description: OGO Shield Web Application Firewall provides protection and performance for your web applications and APIs.
```
Comment (squioc) [inline OGO/ogo-shield/_meta/manifest.yml:4]: A datasource for the format.
```suggestion
description: OGO Shield Web Application Firewall provides protection and performance for your web applications and APIs.

data_sources:
  Web application firewall logs: OGO protects web applications and APIs
```

File: OGO/ogo-shield/ingest/parser.yml [hunk 2]
```diff
@@ -33,7 +33,7 @@ stages:
           tls.cipher: '{{ parser.message.tlsCipher }}'
           tls.version: '{{ parser.message.tlsVersion }}'
           event.action: '{{ parser.message.ogo.appliedAction }}'
-          event.module: ogo.shield
+          event.module: ogo.shield.waf
```
[the remark this answers is not in the document]
Comment (squioc) [reply OGO/ogo-shield/ingest/parser.yml:36]: I will ask our Detection team to add an entry for OGO in this rule

**Review by squioc — CHANGES_REQUESTED** [review]

**Review by squioc — APPROVED** [review]

**Pull request merged**